### PR TITLE
Add SingleColumnValueFilter

### PR DESCRIPTION
**Pull request by hase1031**
Related to https://github.com/alibaba/node-hbase-client/issues/34

Add SingleColumnValueFilter and some Comparators.

### CODE REVIEW

**Comment by fengmk2** [comment]
Cool! Awesome jobs!

Can you fixed the tests?

**Comment by fengmk2** [comment]
I'm going to merge this if the test pass.

**Comment by fengmk2** [comment]
Can you add these filters' usage on README.md?

**Comment by fengmk2** [comment]
Like this https://github.com/alibaba/node-hbase-client#scan-table-and-return-row-key-only .

Given the java code and nodejs code.

**Comment by hase1031** [comment]
OK. I'll add their usages to README.md.

**Comment by fengmk2** [comment]
@hase1031 I write a script to let test run local more easy https://github.com/alibaba/node-hbase-client#run-unit-test

File: test/filters.test.js [hunk 1]
```diff
@@ -64,4 +64,102 @@ describe('test/filters.test.js', function () {
       });
     });
   });
+  describe('BinaryComparator()', function () {
+    describe('write()', function () {
+      it ('should convert BinaryComparator to bytes', function () {
```
Comment (fengmk2) [inline test/filters.test.js]: How about running these filters in a real scan request?

Comment (fengmk2) [inline test/filters.test.js]: Like https://github.com/alibaba/node-hbase-client/blob/master/test/scan.test.js#L51

Comment (fengmk2) [inline test/filters.test.js]: Oh shit, this not real.... too

Comment (fengmk2) [inline test/filters.test.js]: See this https://github.com/alibaba/node-hbase-client/blob/master/test/client.test.js#L730

**Comment by hase1031** [comment]
Sorry, I'll check failed test later ;(

**Comment by fengmk2** [comment]
awesome!

+1

**Comment by hase1031** [comment]
Thanks for your review :)

**Comment by fengmk2** [comment]
release 1.3.0

**Pull request merged**